### PR TITLE
Creating file structure

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**
darkMode

lightMode
componentsScheme1
componentsScheme

### CODE REVIEW

File: src/components/searchBar/citiesSearch/CitiesSearch.jsx [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
-    return (
-        <div>
-            
-        </div>
-    )
+  return <div></div>;
 }
 
-export default CitiesSearch
+export default CitiesSearch;
```
Comment (glebdronchenko) [inline src/components/searchBar/citiesSearch/CitiesSearch.jsx:7]: This _SearchedCities_ name will be better. Your component name is Incomprehensible.

File: src/features/forecatsFeature/components/forecast/currentCityForecastAdditionalInfo/CurrentCityForecastAdditionalInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function CurrentCityForecastAdditionalInfo() {
+  return <div></div>;
+}
+
+export default CurrentCityForecastAdditionalInfo;
```
Comment (glebdronchenko) [inline src/features/forecatsFeature/components/forecast/currentCityForecastAdditionalInfo/CurrentCityForecastAdditionalInfo.jsx:7]: This component name is so long.

File: src/components/searchBar/SearchBar.jsx [hunk 3]
```diff
@@ -1,11 +1,7 @@
...
-    return (
-        <div>
-            
-        </div>
-    )
+  return <div></div>;
 }
 
-export default SearchBar
+export default SearchBar;
```
Comment (glebdronchenko) [inline src/components/searchBar/SearchBar.jsx:7]: Will this component be reused? Maybe _CitySearchBar_ name will be better.

Comment (DmitriyKrivovyaz) [reply src/components/searchBar/SearchBar.jsx:7]: this comonent can be used in different projects that is why i decided to put it inside components

Comment (glebdronchenko) [reply src/components/searchBar/SearchBar.jsx:7]: Ok. Why did you name internal components like _CitiesSearch_ and _SearchedCity_? Maybe these component names would be without _City_ and _Cities_ ?

**Review by ximet — APPROVED** [review]

**Review by glebdronchenko — APPROVED** [review]

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**